src/com/n8pickle/puzzle/Solver.java

```diff
@@ -45,4 +45,12 @@ public static void main(String[] args) {
             StdOut.println("Unsolvable puzzle");
         }
     } // solve a slider puzzle (given below)
+
+    private class SearchNode() implements Comparable<T>{
+        Board b = intitial;
```

Comment:
what is intitial? initial?

Reply:
it is the initial board. I will rename it to fix that